src/Init/Data/BitVec/Bitblast.lean

```diff
@@ -174,6 +174,31 @@ theorem carry_succ (i : Nat) (x y : BitVec w) (c : Bool) :
     exact mod_two_pow_add_mod_two_pow_add_bool_lt_two_pow_succ ..
   cases x.toNat.testBit i <;> cases y.toNat.testBit i <;> (simp; omega)
 
+
+theorem carry_incr (i : Nat) (x : BitVec w) (h : 0 < w) :
```

Comment:
Probably should be called `carry_add_one` or `carry_succ`; we don't use `incr` in names for this.